docs/blog/some-things-to-know-when-developing-with-lit/index.md

```diff
@@ -10,14 +10,13 @@ cover_image: /blog/some-things-to-know-when-developing-with-lit/images/blog-head
 socialMediaImage: /blog/some-things-to-know-when-developing-with-lit/images/social-media-image.png
 ---
 
-When reviewing software with which you have little experience, it's pretty common to attempt to compare it to software you have used before. That can help you get a handle on the general ergonomics and decisions behind the two pieces of software in question. However, one thing that this approach is not particularly good at is comparing the new software _in situ_. You may have spent a good amount of time investigating, researching, and getting comfortable with the software you're already using in the context of your particular use casso much so, that it may even be the best solution available in that context. Comparing a new software in that context (unless it, too, is purpose-built for that context) can do a disservice to the software under test, as well as to your ability to fully understand the benefits of that software.
+When reviewing software with which you have little experience, it's pretty common to attempt to compare it to software you have used before. That can help you get a handle on the general ergonomics and decisions behind the two pieces of software in question. However, one thing that this approach is not particularly good at is comparing the new software _in situ_. You may have spent a good amount of time investigating, researching, and getting comfortable with the software you're already using in the context of your particular use case, so much so that it may even be the best solution available in that context. Comparing a new software in that context (unless it, too, is purpose-built for that context) can do a disservice to the software under test, as well as to your ability to fully understand the benefits of that software.
 
 This is what I keep coming back to when people say things like "X isn't as good as Y" when "X" is a way to build web components ([of which there are many](https://webcomponents.dev/blog/all-the-ways-to-make-a-web-component/)) and "Y" is a JS framework. Even before you fill actual names into either side of the equation the differences in usage of what you fill those in with is important:
 
 - JS frameworks often want to own most or all of a page, whereas a web component is a single custom element on an potentially large and diverse DOM tree.
 - Functional JS frameworks often hide much of that ownership from view to both the benefit (less code) and expense (less flexibility) of developers leveraging them.
 - JS Frameworks tend to be an abstraction above the DOM and in this way, their "components" can exist as both literal (DOM/UI elements) and figurative (data connectivity/translations to non-web contexts).
-- Web components being a DOM element can interact with their position in the DOM in the ways you'd otherwise need to bend over backward to do in a framework.
```

Comment:
I removed it because it seemed like a nearly exact duplicate of the bullet point 2 down from here and that second version reads better to my eyes.  So maybe someone should take a pass at clarifying what different meaning those 2 bullet points are trying to make, but that can't be me.

Reply:
@Westbrook :point_up:

Reply:
Great point!

Reply:
Sorry I didn't call that out in the commit message or here.